src/hevm/src/EVM/Types.hs

```diff
@@ -68,7 +69,7 @@ showAddrWith0x :: Addr -> String
 showAddrWith0x addr = "0x" ++ show addr
 
 showWordWith0x :: W256 -> String
-showWordWith0x addr = show addr
+showWordWith0x = show
```

Comment:
I'd just remove this function and use `show` in its place

Reply:
Shouldn't this rather be smth like:

```
showWordWith0x :: W256 -> String
showWordWith0x w = "0x" ++ show w
```

Reply:
that's already included in the Show instance for W256

Reply:
removed it :slightly_smiling_face: